Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -224,6 +225,43 @@ public class FileHandler: FileHandling {
         try path.throwingGlob(glob)
     }
 
+    public func throwingGlobExcludingOpaqueDirectories(_ path: AbsolutePath, glob: String) throws -> [AbsolutePath] {
+        try throwingGlob(path, glob: glob)
+            .filter { !Self.isInOpaqueDirectory(path: $0.pathString) }
+    }
+
+    static let opaqueDirectoriesExtensions: Set<String> = {
```

Comment:
why can't we declare the `Set` directly instead of having a function that is immediately executed?

Reply:
Will do.